test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -527,5 +534,8 @@ private static async Task SendGitHubPullRequestComments(int prNumber, string bod
 
             response.EnsureSuccessStatusCode();
         }
+
+        private static Policy Retry(int retryCount = 6)
+            => Policy.Handle<Exception>().WaitAndRetry(retryCount, retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryCount)));
```

Comment:
- Are we retrying on all exceptions?
- Why retry 6 times instead of 3 like previously?
- Would `Math.Pow(2, retryCount)` be waiting much longer than before?

Reply:
The maximum waiting time is about 2 mins for each retry, which is about 2 times compared with before.

Reply:
Since the retry is not that common encountered actually, I think retry more times can make a strong promise without reducing the performance a lot....

Reply:
Retry 5 times, and the maximum waiting time is less than 1 min.